WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -255,7 +262,10 @@ private extension OrderListViewController {
                 case .orderCreation:
                     self.setOrderCreationTopBanner()
                 case .IPPFeedback:
-                    self.setIPPFeedbackTopBanner()
+                    guard let survey = self.IPPsurveyVariation else {
+                        return
+                    }
+                    self.setIPPFeedbackTopBanner(survey: survey)
```

Comment:
As above, do you need to guard here?

Reply:
We do seem to need this one, I wonder if is because .sink is expecting a Publisher that never fails, I'm not very comfortable with Combine yet but the compiler is not happy if I remove it. I'll leave it for the moment and do some experimentation on the side 👍